src/components/Login/Login.js

```diff
@@ -0,0 +1,66 @@
+import styled from "styled-components";
+
+function Login() {
+  return (
+    <StyledBackground>
+      <StyledFormat>
+        <section>
+          <StyledLoginMenu>
+            <StyledText>
+              <h3>Welcome! Please Log In</h3>
+
+              <label>User Name</label>
+              <br />
+              <input type="text" id="lname" name="lname" value="" />
+              <br />
+
+              <label>Password</label>
+              <br />
+              <input type="text" id="lname" name="lname" value="" />
+              <br />
+              <FormSubmit type="submit" value="Submit"/>
+            </StyledText>
```

Comment:
You dont need the **`<br />`** here
Just change your **StyledText** to be a _flexbox_ with _direction column_

Reply:
The password input field should look more like this:
`<input type="password" id="password" name="password" value="" />`

Reply:
The `FormSubmit` is actually an input field. Change it to be a button. And you dont need the `type="submit"`

Reply:
The `StyledLoginMenu` is currently a **div**, but I guess it is supposed to be a **form**.

Reply:
What Stefan said 🙂